core/server/admin/controller.js

```diff
@@ -34,5 +33,7 @@ module.exports = function adminController(req, res) {
         });
     }).finally(function noMatterWhat() {
         res.render('default');
-    }).catch(logging.logError);
+    }).catch(function (err) {
+        next(err);
```

Comment:
This code is intended to log any errors from the update check, rather than forward them and cause a crash. Update check shouldn't cause an error?